### DISCUSSION TITLE
Feature request: Debugging Dashboard

### DISCUSSION THREAD

**Discussion by heinemannj** [JensenNick/huawei_solar_pees#63] [Forum]
![image](https://github.com/JensenNick/huawei_solar_pees/assets/3251642/aa3822ae-1bc8-4f11-ab63-59e2a2d3d579)

**My approach**: **To be much close to the nameing conventions provided by @wlcrs huawei_solar** and all other integrations I needs to synchronize:

- huawei_solar
- epexspot
- tibber
- sun
- sun2
- solcast
- emhass
- myVaillant
- dwd_DeutscherWetterdienst
- Shelly
- ...

It works as **visible in above screenshoots**!

For sure tons of bugs and a lot of improvement - but I need clear readable and short sensor names for proper debugging on all integration dependencies ...

**Comment by heinemannj** [comment]
I've made a quick and dirty review of your code.

My view:

- Very complex
- Hardly to maintain
- Not closely enough to @wlcrs naming convention

My conclusion:

- I will step out

**Sorry for the noise ...**

**Comment by JensenNick** [comment]
Not sure what I can do to improve the sensor names. IMO they are as simple as they can be. Did you read the WikiPages? I need some more specific input and/or suggestions - obviously suggestions that are thought through and not random critique.

In the WikiPages I provide some simple debugging dash boards.
